src/Aspire.Dashboard/Configuration/DashboardOptions.cs

```diff
@@ -61,6 +61,11 @@ public sealed class ResourceServiceClientCertificateOptions
     public StoreLocation? Location { get; set; }
 }
 
+public sealed class AllowedCertificateRule
+{
+    public string? Thumbprint { get; set; }
```

Comment:
Wondering why this is nullable. Why would we want an instance of this with a null thumbprint?

If the system was configured with a single rule having a null thumbprint, I don't think any connection would ever be accepted.

X509Certificate2.Thumbprint doens't seem to be nullable either.

Reply:
The property is set by configuration so it could be left out and be null.

But I will add some validation logic for letting the person launching the dashboard that they're made a mistake.